README.md

```diff
@@ -87,6 +89,13 @@ Under the covers Helmfile is simply using the `helm diff` plugin, so that needs
 you should be able to simply execute `helm plugin install https://github.com/databus23/helm-diff`. For more details
 please look at their [documentation](https://github.com/databus23/helm-diff#helm-diff-plugin).
 
+### secrets
+
+The `secrets` parameter in a `helmfile.yaml` causes the [helm-secrets](https://github.com/futuresimple/helm-secrets) plugin to be executed to decrypt the file.
+
+Under the covers Helmfile is simply using the `helm secrets` plugin, so that needs to be installed prior.  For Helm 2.3+
```

Comment:
Although I'm not a native English user, probably:

> Under the covers, Helmfile uses the [helm-secrets](https://github.com/futuresimple/helm-secrets) plugin and therefore you need to install it prior to start using this feature.

would sound a bit more natural(to me 😉).

Reply:
I stole this from the previous block on helm-diff ;)